pkgs/development/misc/resholve/default.nix

```diff
@@ -94,10 +39,16 @@ rec {
     inherit (deps.oil) oildev;
     inherit (deps) configargparse;
     inherit resholve-utils;
+    # used only in tests
+    resholveForTests = resholveForUtils;
```

Comment:
Now that I am thinking about, probably just using `resholve = resholveForUtils` should work.

I will fix this later.

Reply:
I like the fix for the CLI tests broadly (and I agree that `resholve = resholveForUtils` sounds like an improvement).

Reply:
Needed to pass `resholve` as argument for `resholve.nix`, but it is working fine with the change.